admin/travis-build.sh

```diff
@@ -35,6 +35,26 @@ do_lint() {
   make lint
 }
 
+do_pkg() {
+  ./admin/make-macos-pkg.sh --cargo="${HOME}/.cargo/bin/cargo"
+  local pkg="$(echo sandboxfs-*.*.*-????????-macos.pkg)"
+
+  sudo find /Library /etc /usr/local >before.list
+  sudo sysctl -w vfs.generic.osxfuse.tunables.allow_other=0
+
+  sudo installer -pkg "${pkg}" -target /
+  test "$(sysctl -n vfs.generic.osxfuse.tunables.allow_other)" -eq 1
+  /usr/local/bin/sandboxfs --version
```

Comment:
maybe do a equality test here between the `--version` flag value and the pkg filename?